dbtmetabase/_exposures.py

```diff
@@ -214,7 +214,7 @@ def __extract_card_exposures(
                     )
                 elif query_source in ctx.table_names:
                     # Normal question
-                    source_table = ctx.table_names.get(query_source)
+                    source_table = ctx.table_names.get(query_source).lower()
```

Comment:
It could be None, add a similar `if source_table:` check that you saw with `joined_table` below